src/api-reference/common/event-pubsub/travel-search-event.markdown

```diff
@@ -69,16 +71,18 @@ Name|Type|Format|Description
 
 Name|Type|Format|Description
 ---|---|---|---
-`cityName`|`String`|YYYY-MM-DD|Name of the city .
-`cityIATACode`|`String`|YYYY-MM-DD|IATA code of the city.
+`cityName`|`String`|Frankfurt|Name of the city .
+`cityIATACode`|`String`|FRA|IATA code of the city.
 
 ### <a name="schema-hotel-request"></a>Hotel
 
 Name|Type|Format|Description
 ---|---|---|---
 `id`|`String`|GUID|Uniquely identifies the event.
 `correlationId`|String|GUID|Uniquely identifies the hotel search request.
+`eventType`|String|travelSearchHotel|identifies the search event type.|
 `topic`|String|concur.travel.search|
+`subTopic`|String|hotelshop.v1.price|identify sub catagory for hotel search
```

Comment:
The Format here is just confusing -- feel free to schedule some time with me so I can understand what you are trying to accomplish.